src/Client/Client.php

```diff
@@ -36,12 +37,9 @@ public function __construct(GuzzleClientInterface $http)
     }
 
     /**
-     * Create request to send
-     *
-     * @param Request $request
-     * @return RequestInterface
+     * @inheritdoc
      */
-    public function createRequest(Request $request)
+    private function createRequest(Request $request): RequestInterface
```

Comment:
Not sure about making it `private`. I get the idea just I'm not sure if something is not using it. Maybe on tagging change the middle number because of that.